gui/src/connectors/redux/reducers/transactionsIndex.js

```diff
@@ -0,0 +1,22 @@
+const initialState = {
+  currentPage: null,
+  count: 0
+}
+
+export const UPSERT_TRANSACTIONS = 'UPSERT_TRANSACTIONS'
+
+export default (state = initialState, action = {}) => {
+  switch (action.type) {
+    case UPSERT_TRANSACTIONS: {
+      const { data } = action
+      return Object.assign(
+        {},
+        state,
+        { currentPage: data.meta.currentPageTransactions.data.map(t => t.id) },
```

Comment:
Interesting that you have a second reducer just for `transactionsIndex` and its currentPage and count. Why not collapse them into one reducer if they both change on the same action `UPSERT_TRANSACTIONS`?

Reply:
@dimroc I'm trying to move towards separating the entity, domain & ui state https://redux.js.org/recipes/structuring-reducers/normalizing-state-shape#organizing-normalized-data-in-state